### PR TITLE
Fun campus upgrade to richie v2.0.0 beta.14 and release 0.3.0

### PR DESCRIPTION
**Pull request by sampaccoud**
### Added

- Override `course_detail` template to add a message above subheader CTA.

### Changed

- Link the "help" button in menu to a page holding this "reverse id"
- Upgrade richie to 2.0.0-beta.14
- Hide licences block on the course detail pages
- Change aside glimpse title color accordingly to feedback
- Customize filters visible on the search page
- Customize social networks links

### Fixed

- Add i18n messages compilation in the DockerFile so translations are ready

### CODE REVIEW

**Review by jbpenrath — APPROVED** [review]

File: sites/funcampus/src/backend/templates/richie/base.html [hunk 1]
```diff
@@ -15,6 +15,12 @@
 
 {% block userlogin %}{% endblock userlogin %}
 
+{% block topbar_contact %}
+<li class="topbar__item topbar__item--cta">
+    <a href="{% page_url 'help' %}">{% trans "Help" %}</a>
```
Comment (jbpenrath) [inline sites/funcampus/src/backend/templates/richie/base.html:20]: You forgot to load `cms_tags`

**Review by jbpenrath — CHANGES_REQUESTED** [review]
You go

File: sites/funcampus/src/backend/templates/richie/base.html (hunk 1 above)
Comment (sampaccoud) [reply sites/funcampus/src/backend/templates/richie/base.html:20]: Well spotted :clap:

**Review by jbpenrath — APPROVED** [review]

**Comment by sampaccoud** [comment]
@jbpenrath while test-driving the funcorporate site, I noticed some missing scss following the richie upgrade. I  added it to this PR for funcampus.

**Pull request merged**